New package: EphemerisSources v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: EphemerisSources
- Repository: https://github.com/cadojo/EphemerisSources.jl
- Created by: @cadojo
- Version: v0.1.0
- Commit: 37381c4cf9fab6d9708c92d3328eb5ffd5db17f8
- Reviewed by: @cadojo
- Reference: https://github.com/cadojo/EphemerisSources.jl/commit/37381c4cf9fab6d9708c92d3328eb5ffd5db17f8#commitcomment-144459314
- Description: Access HORIZONS and SPICE ephemeris sources with idiomatic Julia.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6131c4c22b4242009585c48ce57f6d964bc054adfd91146c3348256fa67e870a253930c47b4f601be86d025ce70b35f144120a178d131d87d5d765f3e288571d845e27f064ec5419c60c9777e064c64b0052c42060609da42253baf796556a7ba481beed7f0772a5e8d7adb377b54e153f4ba380fc2bc1374dc175a8d5d6db45ffecc222809c7feb7691e3f201155f02746117c0dc862727d582f318bdc0b107b97cf0c213d4f63ed10aea733153f0d38 -->

cadojo commented:
This PR is the result of an invalid @JuliaRegistrator syntax: `@JuliaRegistrator register "lib/SPICEBodies`. Please close this PR.

JuliaRegistrator committed:
New package: EphemerisSources v0.1.0

UUID: 858900df-f7ae-402a-aa8c-3e4ef85aa271
Repo: https://github.com/cadojo/EphemerisSources.jl.git
Tree: 50f2a3ea172a6b2d67b4baa6b9c85f8ae256c047

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
